inspirehep/modules/workflows/workflows/article.py

```diff
@@ -310,26 +310,35 @@
             is_marked('auto-approved'),
             mark('approved', True),
             [
-                IF_NOT(
+                IF_ELSE(
                     is_record_relevant,
```

Comment:
this is only one case. There's also the case where the record gets manually rejected by a curator (where it should be added to the private collection instead and go through all the same workflow steps).

Reply:
Yeah, you are right. Fixed now and I added additional test to check this.